duckduckgo/iOS
# Improve cache line on tab-switcher

Reviewer: Caine
Asana: https://app.asana.com/0/361428290920652/386683237338087

**Description**:
Update cached text to be more human friendly. Also updated it to only display when we initially enter the tabs switcher if we have cached data

**Steps to test this PR**:
1. Open the app, browse to a webpage
2. Open the tab switcher and observe new text
3. Press the forget button and note that the text disappears

###### Reviewer Checklist:


###### PR DRI Checklist:
